### ISSUE TITLE
TestBalancer_roundrobin_with_weight failed

### ISSUE THREAD

**Issue by juzhiyuan** [apache/apisix-dashboard#1401]
fine, I don't mean CI failed due to this PR 😂 just saw after PR merged, CI down.

![image](https://user-images.githubusercontent.com/2106987/106355876-d69d9480-6335-11eb-9917-0c9b752dcd17.png)

_Originally posted by @juzhiyuan in https://github.com/apache/apisix-dashboard/issues/1396#issuecomment-770201476_

**Comment by imjoey** [comment]
FYI, this failure began from this morning, and 100% reproducible.

**Comment by Jaycean** [comment]
I just ran the local test file balancer_ test.go. There is no problem with it. This problem occurred this morning, which made PR CI unable to pass.

**Comment by Jaycean** [comment]
In addition, in the development process, I also feel that there are many CI test cases, and the error information is very vague. When there are errors in the test cases, it also makes it more difficult for us to locate the problem. Of course, it takes a lot of time to improve this thing.

**Comment by Jaycean** [comment]
@juzhiyuan Hi,PR #1395 has fixed the problem. The reason for the problem is apisix has modified luajit. @nic-chen has fixed the problem.

**Comment by imjoey** [comment]
> @juzhiyuan Hi,PR #1395 has fixed the problem. The reason for the problem is apisix has modified luajit. @nic-chen has fixed the problem.

@Jaycean yep, while we would better see the CI result triggered by https://github.com/apache/apisix-dashboard/commit/94d952f9a43a9d6507b2966b58791f789e5b74a6 push event before close this issue.

**Comment by juzhiyuan** [comment]
https://github.com/apache/apisix-dashboard/runs/1804604649#step:4:703

failed due to luasec.

**Comment by imjoey** [comment]
> https://github.com/apache/apisix-dashboard/runs/1804604649#step:4:703
> 
> failed due to luasec.

@juzhiyuan yep, this may cause be the lua patch, related to https://github.com/apache/apisix/pull/3335 authored by @fukiki . We are trying to fix this. 😄

**Comment by Jaycean** [comment]
@juzhiyuan Hi,The problem has been solved. Since APISIX has modified Luajit to cause dependency environment problems, the dockfile has been modified to fix it. As mentioned in PR, pulling the dockerfile file of remote apifix should open a new issue discussion.This issue can be closed.

**Comment by nic-chen** [comment]
thanks @Jaycean
